[2025-06-29T12:17:34.537+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-06-18T00:00:00+00:00 [queued]>
[2025-06-29T12:17:34.706+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-06-18T00:00:00+00:00 [queued]>
[2025-06-29T12:17:34.710+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-06-29T12:17:34.978+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-06-18 00:00:00+00:00
[2025-06-29T12:17:35.015+0000] {standard_task_runner.py:57} INFO - Started process 8284 to run task
[2025-06-29T12:17:35.110+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2023-06-18T00:00:00+00:00', '--job-id', '2097', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpcddiy0v0']
[2025-06-29T12:17:35.172+0000] {standard_task_runner.py:85} INFO - Job 2097: Subtask insert_into_table
[2025-06-29T12:17:35.992+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-06-18T00:00:00+00:00 [running]> on host f561bfe7dd3a
[2025-06-29T12:17:36.845+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-18T00:00:00+00:00'
[2025-06-29T12:17:36.858+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-06-18', 'dag_with_postgres_operator_v03')
        
[2025-06-29T12:17:36.915+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T12:17:36.983+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T12:17:37.016+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-06-18', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-06-29T12:17:37.018+0000] {sql.py:427} INFO - Rows affected: 1
[2025-06-29T12:17:37.080+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20230618T000000, start_date=20250629T121734, end_date=20250629T121737
[2025-06-29T12:17:37.225+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-29T12:17:37.611+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
